convert2rhel/toolopts.py

```diff
@@ -65,15 +66,20 @@ def _get_argparser():
                  "  convert2rhel [-k key] [-o organization] [--pool pool_id |"
                  " -a] [--disablerepo repoid] [--enablerepo repoid] [-v"
                  " variant] [--serverurl url] [--no-rpm-va] [--debug]"
-                 " [--restart] [--disable-colors] [-y]")
+                 " [--restart] [--disable-colors] [-y]"
+                 "\n\n"
+                 "WARNING: The tool needs to be run under the root user"
+                 )
         return optparse.OptionParser(conflict_handler='resolve',
                                      usage=usage,
-                                     add_help_option=False)
+                                     add_help_option=False,
+                                     version=__version__)
 
     def _register_options(self):
         """Prescribe what command line options the tool accepts."""
         self._parser.add_option("-h", "--help", action="help", help="Show "
                                 " help message and exit.")
+        self._parser.add_option('--version', action='version', help="Show version of the application.")
```

Comment:
```suggestion
        self._parser.add_option('--version', action='version', help="Show version.")
```
Using _"application"_ would be inconsistent with the rest of the messages where we use _"tool"_. Even in this PR you're adding _"WARNING: The tool needs to be run under the root user"_ :)

I'd rather see us using _"utility"_ instead of _"tool"_ but let's keep it as tool for now.

Reply:
Haha, @SpyTec was faster.

Reply:
ok:)